flickr.py

```diff
@@ -25,13 +25,14 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(tags, method = "flickr.photos.search", photo_id = None):
+
     if not FLICKR_API_KEY:
```

Comment:
If your you try to use 'flickr.photo.getInfo', this function does not work here.
I think you should write a more general function for search.